unittests/runtime-ffi/ffi.cpp

```diff
@@ -312,6 +312,33 @@ BOOST_AUTO_TEST_CASE(call) {
   ret = *(int *) bytes->data;
 
   BOOST_CHECK_EQUAL(ret, p2.p.x * p2.p.y);
+
+  /* int pointerTest(int * x) */
+  x = 2;
+  int * ptr = &x;
+  string * ptrargstr = makeString((char *) &ptr, sizeof(int *)); 
+
+  block * ptrarg = static_cast<block *>(koreAlloc(sizeof(block) + sizeof(string *)));
```

Comment:
can we actually call hook_FFI_alloc, get a string *, write to it, call hook_FFI_bytes_address, call mpz_get_ui, write the value to the argument to call, then call, then call hook_FFI_free? I'd like to test this use case a bit more explicitly.